README.md

```diff
@@ -27,7 +28,11 @@ Proudly brought to you by [@revolunet](http://twitter.com/revolunet), [@deltaeps
  - offline mode
  - analytics.js advanced debugging support
 
-## Installation
+## Installation and Quick Start
+You need to read rest of documentation, but here is the quick start.
+
+#### 1- Installation:
```

Comment:
Why use a #4 header instead of a #3 here? This section is under a #2 header, so it would make sense that it is a #3.